dev/docker/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN curl --show-error https://getcomposer.org/installer | php -- \
 RUN useradd -ms /bin/sh app
 
 # Enable parallel package installation for Composer
-RUN su-exec app composer global require hirak/prestissimo
```

Comment:
Flex is also installing packages in parallel and it supports specifying symfony versions